feature-detects/css/multiplebgs.js

```diff
@@ -6,17 +6,17 @@
   "tags": ["css"]
 }
 !*/
-define(['Modernizr', 'createElement'], function(Modernizr, createElement) {
+define(['Modernizr', 'setCss', 'createElement'], function(Modernizr, setCss, createElement) {
   // Setting multiple images AND a color on the background shorthand property
   // and then querying the style.background property value for the number of
   // occurrences of "url(" is a reliable method for detecting ACTUAL support for this!
 
   Modernizr.addTest('multiplebgs', function() {
-    var style = createElement('a').style;
-    style.cssText = 'background:url(https://),url(https://),red url(https://)';
+    var elem = createElement('a');
```

Comment:
```suggestion
    var elem = createElement('div');
```
The other PR was creating a <div> here - but that wasn't their change, so I kept the <a> element from main branch.  I honestly don't know why they would be so specific with the element, though